eng/native/configureplatform.cmake

```diff
@@ -169,6 +169,11 @@ endif()
 # if target arch is not specified then host & target are same
 if(NOT DEFINED CLR_CMAKE_TARGET_ARCH OR CLR_CMAKE_TARGET_ARCH STREQUAL "" )
   set(CLR_CMAKE_TARGET_ARCH ${CLR_CMAKE_HOST_ARCH})
+
+  if (CLR_CMAKE_HOST_ARCH_ARMV7L)
+    set(CLR_CMAKE_TARGET_ARCH_ARMV7L 1)
+    set(ARM_SOFTFP 1)
+  endif()
```

Comment:
These lines shouldn't be needed.  Line 190 below should be sufficient.  Any reason this is duplicated here?

Reply:
I've updated my commit, this would be a better solution. Problem was that we ended up with
```
CLR_CMAKE_HOST_UNIX_ARM 1
CLR_CMAKE_HOST_UNIX_ARMV7L 1
CLR_CMAKE_HOST_ARCH_ARM 1
CLR_CMAKE_HOST_ARCH "arm"
CLR_CMAKE_HOST_ARCH_ARMV7L 1
```

which lead to
```
CLR_CMAKE_TARGET_ARCH "arm"
CLR_CMAKE_TARGET_ARCH_ARM 1
```

Now, `CLR_CMAKE_HOST_ARCH "armel"` and target will be set correctly.

Reply:
Do you need `set(ARM_SOFTFP 1)` here to?  From your previous discussion, I assume Line 191 below will not be reached.

Reply:
ARM_SOFTFP=1 is passed from outside for this case (see `./eng/native/gen-buildsys.sh`), so it is not needed here. 
We can't set neither CLR_CMAKE_TARGET_ARCH to "armel", nor CLR_CMAKE_HOST_ARCH, because we actually can't distinguish armel from arm here (CLR_CMAKE_HOST_ARCH_ARMV7L=1 for both).